libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -285,6 +253,11 @@ class AP_AHRS
     // if we have an estimate
     virtual bool airspeed_estimate(float *airspeed_ret) const WARN_IF_UNUSED;
 
+    // airspeed helper used by scripting
+    bool get_airspeed(float &air_speed) const {
```

Comment:
better to fix the airspeed_estimate() to take a reference

Reply:
fixed